### PR TITLE
Reconfigure yarn dev and .env

### PR DESCRIPTION
**Pull request by kachkaev**
This PR follows the today’s conversation with @saifulkhan and @phongvis.

- We use `yarn dev` as a single entry point for running the local dev server.
- We use production API endpoints by default (this can change later if we introduce staging server). UI devs end up with s simpler onboarding process.
- To switch to local API servers, a developer should create `.env.local` and define `localhost` URLs. The file is not tracked by git, unlike `.env`. This approach matches [Next.js docs](https://nextjs.org/docs/basic-features/environment-variables).

Out of scope:
- Switching from port 5000 to 3000 (we need to change the default API server port first).
- Configuring `yarn build`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,8 +2,7 @@
   "private": true,
   "scripts": {
     "build": "next build",
-    "dev-prod": "env-cmd -f .env.production next dev",
-    "dev": "env-cmd -f .env.development next dev",
+    "dev": "next dev --port 5000",
```
Comment (kachkaev) [inline package.json:5]: Next.js does not read `PORT` environment variable from `.env` out of box. The argument will be soon removed anyone, once the API endpoint is switched to a different port (e.g. 4000...4999)

**Comment by saifulkhan** [comment]
You can change the rampvis-ui port no. to 3000, and the APIs to 4000, 4001. I shall update the APIs.

**Comment by kachkaev** [comment]
@saifulkhan shall we do this in a separate PR / commit just to keep the scope small. Feel free to approve the PR if you agree, otherwise happy to push a commit with the change before we merge this PR.

File: .env [hunk 2]
```diff
@@ -1,4 +1,2 @@
-PORT=5000
-NEXT_PUBLIC_API_PY=http://vis.scrc.uk/stat/v1
 NEXT_PUBLIC_API_JS=http://vis.scrc.uk/api/v1
-NEXT_PUBLIC_UI_URL=http://vis.scrc.uk
\ No newline at end of file
+NEXT_PUBLIC_API_PY=http://vis.scrc.uk/stat/v1
```
Comment (kachkaev) [inline .env:2]: I swapped the two variables so that the list was alphabetically ordered

**Pull request merged**